src/documents/TextfieldAPI.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Textfield
+layout: default
+category: Components
+---
+
+The **Textfield** is a small component for adding aria attributes and data binding to a HTML textfield. Typically this will be used in conjunction with a controller (e.g. [**Textfield Slider**](TextfieldSlider.md), [**Textfield Stepper**](TextfieldStepper.md)) to restrict the values and pair with additional input methods.
```

Comment:
Filenames for TextfieldSlider should be `TextfieldSliderAPI.md`. Same applies to TextfieldStepper.